src/main/java/tornadofx/CSS.kt

```diff
@@ -953,13 +952,69 @@ class CssSelectionBlock(op: CssSelectionBlock.() -> Unit) : PropertyHolder(), Se
     @Deprecated("Use and() instead as it's clearer", ReplaceWith("and(selector, *selectors, op = op)"))
     fun add(selector: Selectable, vararg selectors: Selectable, op: CssSelectionBlock.() -> Unit) = and(selector, *selectors, op = op)
 
-    fun and(selector: String, op: CssSelectionBlock.() -> Unit) = and(selector.toSelector(), op = op)
-    fun and(selector: Selectable, vararg selectors: Selectable, op: CssSelectionBlock.() -> Unit): CssSelection {
+    private fun addRelation(
+        relation: CssSubRule.Relation,
+        selector: Selectable, selectors: Array<out Selectable>,
```

Comment:
This should be `selector: Selectable, vararg selectors: Selectable` and you would call this function with `addRelation(<relation>, selector, *selectors, op = op)`

Reply:
Is there any reason to use additional spread operator?

Reply:
Yes. For one thing, we've been trying to push using idiomatic code throughout the codebase. For another, it makes the intended use clear and ensures consistency,